tests/test_syntax.py

```diff
@@ -74,6 +89,21 @@ def test_python_render_simple():
     assert rendered_syntax == expected
 
 
+def test_python_render_simple_passing_lexer_instance():
+    syntax = Syntax(
+        CODE,
+        lexer=PythonLexer(),
+        line_numbers=False,
+        theme="foo",
+        code_width=60,
+        word_wrap=False,
+    )
+    rendered_syntax = render(syntax)
+    print(repr(rendered_syntax))
```

Comment:
```suggestion
```

?

Reply:
uh, good spot :D I've copied this from the test below! maybe @willmcgugan knows if we should remove it or not (if so I can remove it from all the tests in this file)

Reply:
I sometimes leave those in tests, so that pytest shows the full output if there is a failure.

I don't mind either way.

Reply:
makes sense! I'll leave it then 😊